### PR TITLE
Color picker maybe

### PR DESCRIPTION
**Pull request by neferent**
Rewrote color picker for typescript,

seems to work now,

color type lowered to HEX only.

$emits hex code.

### CODE REVIEW

**Comment by neferent** [comment]
fixed conflict

**Comment by neferent** [comment]
fixed another conflict

File: src/components/ColorPicker.vue [hunk 1]
```diff
@@ -89,31 +86,14 @@ export default {
   .radius();
   box-sizing: border-box;
   position: absolute;
-  top: 7px;
+  top: 10px;
   right: 8px;
-  border: 1px solid @day-input-border;
+  border: 1px solid rgba(9, 22, 29, 0.12);
```
Comment (mbiemiller) [inline src/components/ColorPicker.vue]: this should be @day-input-border
was it changed for a reason?

File: src/components/ColorPicker.vue [hunk 2]
```diff
@@ -15,61 +13,60 @@
...
+  private backgroundColor: String = "";
+  private color: String = this.defaultColor;
+
+  @Prop()
+  value!: any;
+
+  updateFromPicker(value: any) {
+    this.color = value.hex;
+    this.$emit("input", this.color);
+    console.log(this.color);
```
Comment (mbiemiller) [inline src/components/ColorPicker.vue]: remove console log

**Pull request merged**